platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -603,14 +407,21 @@ public class SwirldsPlatform implements Platform, ConnectionTracker, Startable {
                 settings.getEventStreamQueueCapacity(),
                 this::isLastEventBeforeRestart);
 
-        if (chatterConfig.useChatter()) {
-            criticalQuorum = new CriticalQuorumImpl(
-                    metrics, selfId.id(), initialAddressBook, false, chatterConfig.criticalQuorumSoftening());
+        final EventTaskDispatcher taskDispatcher;
```

Comment:
The declarations for `taskDispatcher` and `eventObserverDispatcher` would be better placed down where `preConsensusEventHandler` is declared, since these final variables are set in that try block.

Reply:
Fixed. There is actually no need for the declaration and assignment to not be in the exact same place any more. This is a holdover from before the refactor where a bunch of stuff was defined in a bunch of places other than the constructor.